design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -143,6 +143,42 @@ The proposed API changes for VM objects can be seen in
 [the VM API examples section](#vms), while the proposed API changes for the VMI
 object can be seen in [the VMI API examples section](#vmis).
 
+### Pod interface naming
+As of today, the VMs’ pod interfaces names has an ordinal factor 
+(`net1`, `net2`, …, `netX`) which is derived from the specified in the VMI spec,
+the interfaces names are specified in the virt-launcher pod 
+`k8s.v1.cni.cncf.io/networks` annotation, and eventually the pod is created with 
+those interface names.
```

Comment:
```suggestion
The VMs’ pod interface names are ordinal based, derived from their order in the VMI spec;
they are requested from Multus by specifying any additional networks using the `k8s.v1.cni.cncf.io/networks` annotation on the launcher pod (which is templated by KubeVirt).
```

I guess this could still be improved, but, I also think it makes a decent job at getting the message across.